docs/csharp/tour-of-csharp/arrays.md

```diff
@@ -5,7 +5,7 @@ ms.date: 02/27/2020
 ms.assetid: a440704c-9e88-4c75-97dd-bfe30ca0fb97
 ---
 
-# Arrays
+# Arrays in C\#
```

Comment:
Do you want me to add this suffix to all articles in *tour-of-csharp* for consistency?

Reply:
No, I don't think this is common enough.